textworld/utils.py

```diff
@@ -147,7 +147,7 @@ def make_temp_directory(suffix='', prefix='tmp', dir=None):
     """ Create temporary folder to used in a with statement. """
     temp_dir = tempfile.mkdtemp(suffix, prefix, dir)
     try:
-        yield temp_dir
+        yield os.path.join(temp_dir, "")  # So path ends with '/'.
```

Comment:
What's this needed for?

Reply:
It's needed for [compile_game](https://github.com/Microsoft/TextWorld/blob/master/textworld/generator/__init__.py#L217). If a folder is provided as the path, we use the game's UUID for the filename. To detect if it is a folder (might not exists yet), I essentially check if it ends with a "/". So, as a convenience, I've added it to the `make_temp_directory`.